src/main/webapp/i18n/en/lti.json

```
@@ -0,0 +1,18 @@
+{
+    "artemisApp": {
+        "lti": {
+            "initializationError": "An user initialization progress was started but either the user was not marked or is not an LTI user. Please ask your instructor for help if you didn't receive your Artemis password yet.",
```

```suggestion
            "initializationError": "A user initialization progress was started but either the user was not marked or is not an LTI user. Please ask your instructor for help if you didn't receive your Artemis password yet.",
```
Same comment as with the German translation applies here as well.